cypress/features/regression/experimental/radioButtonGroup.feature

```diff
@@ -28,4 +28,90 @@ Feature: Experimental RadioButtonGroup component
+
+  @positive
+  Scenario Outline: Change label to <label>
+    When I open default "Experimental RadioButton" component in noIFrame with "radioButtonGroup" json from "experimental" using "<nameOfObject>" object name
+    Then "first" radioButton on preview is <label>
+    Examples:
+      | label                        | nameOfObject          |
+      | mp150ú¿¡üßä                  | labelOtherLanguage    |
+      | !@#$%^*()_+-=~[];:.,?{}&"'<> | labelSpecialCharacter |
+
+  @positive
+  Scenario Outline: Change labelHelp to <labelHelp>
+    When I open default "Experimental RadioButton" component in noIFrame with "radioButtonGroup" json from "experimental" using "<nameOfObject>" object name
+      And I hover mouse onto "first" help icon in NoIFrame
+    Then tooltipPreview on preview in noIframe is set to <labelHelp>
+    Examples:
+      | labelHelp                    | nameOfObject              |
+      | mp150ú¿¡üßä                  | labelHelpOtherLanguage    |
+      | !@#$%^*()_+-=~[];:.,?{}&"'<> | labelHelpSpecialCharacter |
+
+  @positive
+  Scenario Outline: Change weekly value to <weeklyValue>
+    When I open default "Experimental RadioButton" component in noIFrame with "radioButtonGroup" json from "experimental" using "<nameOfObject>" object name
+    Then "first" RadioButton has value <weeklyValue>
+    Examples:
+      | weeklyValue                  | nameOfObject                |
+      | mp150ú¿¡üßä                  | weeklyValueOtherLanguage    |
+      | !@#$%^*()_+-=~[];:.,?{}&"'<> | weeklyValueSpecialCharacter |
+
+  @positive
+  Scenario: Disable RadioButton
+    When I open default "Experimental RadioButton" component in noIFrame with "radioButtonGroup" json from "experimental" using "disabled" object name
+    Then "first" RadioButton component is disabled
+
+  @positive
+  Scenario: Disable and enable RadioButton
+    When I open default "Experimental RadioButton" component in noIFrame with "radioButtonGroup" json from "experimental" using "disabledFalse" object name
+    Then "first" RadioButton component is enabled
+
+  @positive
+  Scenario: Enable reverse RadioButton
+    When I open default "Experimental RadioButton" component in noIFrame with "radioButtonGroup" json from "experimental" using "reverse" object name
+    Then "first" RadioButton is set to reverse
+
+
+  @positive
+  Scenario: Disable reverse RadioButton
+    When I open default "Experimental RadioButton" component in noIFrame with "radioButtonGroup" json from "experimental" using "reverseFalse" object name
+    Then "first" RadioButton is not set to reverse
+
+  @positive
+  Scenario Outline: Change RadioButton size to <size>
+    When I open default "Experimental RadioButton" component in noIFrame with "radioButtonGroup" json from "experimental" using "<nameOfObject>" object name
+    Then "first" RadioButton size on preview is set to "<size>"
+    Examples:
+      | size  | nameOfObject |
+      | small | smallSize    |
```

Comment:
we are using naming like: ${prop} ${value}. It should be like `sizeSmall` and `sizeLarge`

Reply:
fixed